examples/tests/backend_driver.py

```diff
@@ -401,13 +123,13 @@ def drive(backend, directories, python=['python'], switches=[]):
             'from pylab import savefig\n',
             'import numpy\n',
             'numpy.seterr(invalid="ignore")\n',
-            ))
+        ))
         for line in open(fullpath):
             line_lstrip = line.lstrip()
-            if (line_lstrip.startswith('from __future__ import') or
-                    line_lstrip.startswith('matplotlib.use') or
-                    line_lstrip.startswith('savefig') or
-                    line_lstrip.startswith('show')):
+            if ('from __future__ import' in line_lstrip or
```

Comment:
The `lstrip` is not needed now that you're using `in`, but also, this can be re-written a bit:
```
skippable = [
    'from __future__ import',
    'matplotlib.use',
    'plt.savefig',
    'plt.show',
]
if any(skip in line for skip in skippable):
```
(choose better names)